easybuild/easyconfigs/p/protobuf/protobuf-3.1.0-CrayGNU-17.08-python3.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = 'PythonPackage'
+
+name = 'protobuf'
+version = '3.1.0'
+
+py_maj_ver = '3'
```

Comment:
What do you think about using only the `req_py_majver` and `req_py_minver`?

Reply:
Actually I have thought about this as well, but then I have convert to str everytime I use it, and that is quite often. Definitely decreases code readability.

On the other hand, req_py_* is only used internally.

Reply:
👍